package.json

```diff
@@ -52,15 +55,20 @@
         "@rollup/plugin-babel": "^5.0.2",
         "@rollup/plugin-commonjs": "^12.0.0",
         "@rollup/plugin-node-resolve": "^8.0.0",
+        "@typescript-eslint/eslint-plugin": "^4.11.0",
+        "@typescript-eslint/parser": "^4.11.0",
         "eslint": "^7.1.0",
         "eslint-config-hive": "^0.3.3",
+        "fastify": "^3.9.2",

```

Comment:
I believe this qualifies to dependency and not dev dependency

Reply:
Given this is only used for types I think it can be a dev dependency. No?